Labs AI Kapuler Kirill KI-20-02

kapulerDNY opened the pull request.

kapulerDNY committed:
Create readme.txt

kapulerDNY committed:
Add files via upload

kapulerDNY committed:
Update README.md

kapulerDNY committed:
Delete lab01/readme.txt

kapulerDNY committed:
Update main.py

Почти полностью изменил код , взял базу 1 версий. Перепроверил условия задачи. Где увидел что можно использовать библиотечные методы , использовал.

kapulerDNY committed:
Create 02

kapulerDNY committed:
Create main.py

kapulerDNY committed:
Update README.md

ffeks-dnu-edu-ua commented on lab02/main.py:
```diff
@@ -0,0 +1,30 @@
+toolbox.register("select", tools.selTournament, tournsize=3)
+
+population = toolbox.population(n=50)
+algorithms.eaMuPlusLambda(population, toolbox, mu=20, lambda_=100, cxpb=0.7, mutpb=0.2, ngen=100)
```
спростили собі задачу?)

ffeks-dnu-edu-ua commented on lab01/main.py:
```diff
@@ -0,0 +1,90 @@
+
+# 1. методом зсуву середнего визначає кількість кластерів розбиття
+def meanShift(data):
+    bandwidth = MeanShift(bandwidth=None)
```
у вас не працює цей код без bandwidth

ffeks-dnu-edu-ua reviewed (changes requested):
check comments

kapulerDNY committed:
Create test.txt

kapulerDNY committed:
Create main.py

kapulerDNY committed:
Delete lab03/test.txt

kapulerDNY committed:
Add files via upload

kapulerDNY committed:
Create New_Version.py

Версия без использование подбора данных и последующего получения меток кластеров и центров кластеров тобишь bandwidth

kapulerDNY committed:
Update README.md

Skuratovskyi commented on lab01/New_Version.py:
```diff
@@ -0,0 +1,74 @@
+from sklearn.metrics import silhouette_score
+
+def meanShift(data): #Ушел от bandwidth 
+    meanshift = MeanShift()
```
так а навіщо ви видалили bandwidth взагалі?
я ж писав 
```
# Estimate the bandwidth of X
bandwidth_X = estimate_bandwidth(X, quantile=0.14, n_samples=len(X))
# Cluster data with MeanShift
meanshift_model = MeanShift(bandwidth=bandwidth_X, bin_seeding=True)
```

Skuratovskyi reviewed (approved):
знову мимо.
